### DISCUSSION TITLE
Dark Mode for PyWebView

### DISCUSSION THREAD

**Discussion by byte127x** [r0x0r/pywebview#1174] [Q&A]
In my current app, the titlebar looks light
https://i.imgur.com/QyI9Ifx.png

Is there any way to get it to look dark, like it does in this other programs?
https://i.imgur.com/eJqG5h8.png

(Using Windows 11, Python 3.11, pywebview 4.2.2)

**Comment by byte127x** [comment]
is there no way yet?

**Comment by byte127x** [comment]
@r0x0r can you add something like this?

**Comment by r0x0r** [comment]
Dark mode is not implemented, but it could be done. Contributions welcomed.

**Comment by nickbutsky** [comment]
If you are using winforms you can do this:
```Python
import ctypes
from ctypes import wintypes

import webview
from webview.platforms.winforms import BrowserView

window = webview.create_window("")

dwmapi = ctypes.windll.LoadLibrary("dwmapi")
window.events.shown += lambda: dwmapi.DwmSetWindowAttribute(
    BrowserView.instances[window.uid].Handle.ToInt32(),
    20,
    ctypes.byref(ctypes.c_bool(True)),
    ctypes.sizeof(wintypes.BOOL),
)
webview.start()
```
When using a different renderer you'll have to find some other way to get the window handle.

**Reply by EmberLightVFX** [reply]
Works great! But when the window is selected the bars color becomes black instead of grey like when it's de-selected.
Comparing to other windows the bar color stays grey and only the text becomes white.
I have no idea what the above code actually does but is it something in there that decides the selected-color?
